gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -610,13 +675,35 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
           goto error;
         }
       }
-#if 0
-      /** @todo Need to updatd out_tensors */
-      /* get loss, accuracy, val_loss, val_accuracy */
-      double data[4] = { 0, 0, 0, 0 };
+
+      ret =
+          trainer->fw->getStatus (trainer->fw, &trainer->prop,
+          trainer->privateData);
+      if (ret < 0) {
+        GST_ERROR_OBJECT (trainer,
+            "Faild to get stats from the model being trained in subplugin(%s).",
```

Comment:
The word 'stats' (especially, when used with the model) has the possibility to mislead. How about changing the error message to be more clear?

Reply:
Thank you.